composer.json

```diff
@@ -41,10 +41,15 @@
         "whitehat101/apr1-md5": "~1.0",
         "twig/twig": "~1.0 || ~2.0",
         "gettext/gettext": "^4.6",
-        "jaimeperez/twig-configurable-i18n": "^2.0"
+        "jaimeperez/twig-configurable-i18n": "^2.0",
```

Comment:
Should we move this to simplesamlphp namespace or something else with shared responsibility? As to prevent blockers if one person is not available.